plugins/backend/backstage-plugin-argo-cd-backend/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@roadiehq/backstage-plugin-argo-cd-backend",
+  "version": "1.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.9.6",
+    "@backstage/catalog-client": "^0.5.0",
+    "@backstage/config": "^0.1.8",
+    "@types/express": "^4.17.6",
+    "axios": "^0.21.1",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
+    "node-fetch": "^2.6.1",
```

Comment:
Looks like `axios` is used as the XHR client for all request so you should be able to leave this out. Same for the global fetch definition in `plugins/backend/backstage-plugin-argo-cd-backend/src/setupTests.ts`

Reply:
addressed